query/test/functional/test_endpoints.py

```diff
@@ -30,6 +30,48 @@ def test_query(self):
         self.assertEqual(json.loads(response)['query'], query)
         self.assertGreaterEqual(json.loads(response)['results'][0]['count'], 0)
 
+    def test_create_long_running_query(self):
```

Comment:
You should probably change this to reflect your name change to `create_async_query`. There's a bunch more occurrences of `long_query` and `long_running_query` in the codebase. I'd do a text search and make all of them consistent.